besom-cfg/lib/src/main/scala/Configured.scala

```diff
@@ -3,6 +3,11 @@ package besom.cfg
 import scala.quoted.*
 import besom.json.*
 import besom.cfg.internal.*
+import besom.util.Validated
+import scala.util.control.NoStackTrace
+import org.checkerframework.checker.units.qual.A
+
+final val Version = "0.4.0-SNAPSHOT"
```

Comment:
Yet another place to remember about when bumping versions 😞 
How about using scala-cli's `BuildInfo` to keep this up to date with what we have in the module's build definition?

Reply:
yeah but no, we're moving away from macros, right?